backend/ethereum/wallet/address.go

```diff
@@ -43,22 +40,14 @@ func (a *Address) MarshalBinary() ([]byte, error) {
 	return (*common.Address)(a).Bytes(), nil
 }
 
-// Decode decodes an address from a io.Reader. Part of the
-// go-perun/pkg/io.Serializer interface.
-func (a *Address) Decode(r io.Reader) error {
-	var length int64
-	err := perunio.Decode(r, &length)
-	if err != nil {
-		return err
-	}
-	if length != common.AddressLength {
-		return fmt.Errorf("unexpected address length %d, want %d", length, common.AddressLength)
+// UnmarshalBinary unmarshalled the address from its binary representation.
+func (a *Address) UnmarshalBinary(data []byte) error {
+	if len(data) != common.AddressLength {
+		return fmt.Errorf("unexpected address length %d, want %d", len(data), 64)
```

Comment:
Use of magic number. Linter should be complaining!?

Reply:
Updated in c3716d51027f23468796c574881805e90d3c6ac6.